metrics/web/lib/common/presentation/navigation/state/navigation_notifier.dart

```diff
@@ -215,7 +215,10 @@ class NavigationNotifier extends ChangeNotifier {
 
     _currentConfiguration = newConfiguration;
 
-    final newPage = _pageFactory.create(_currentConfiguration);
+    final newPage = _pageFactory.create(
+      _currentConfiguration.name,
+      _currentPageParameters,
+    );
```

Comment:
✅ 
Should we test that the `.push` and `.handleInitialRoutePath` methods use the given page factory to create a page? Also, I believe, we should test that these methods provide the correct parameters to the `MetricsPageFactory.create` method.